build.gradle.kts

```diff
@@ -15,6 +15,11 @@ description = "Adaptable, fast automation for all"
 dependencyAnalysis {
     issues {
         all {
+            onUnusedAnnotationProcessors {
+                // Ignore check for internal-instrumentation-processor, since we apply
+                // it to all distribution.api-java projects but projects might not have any upgrade
+                exclude(":internal-instrumentation-processor")
```

Comment:
💭 We need to exclude `:internal-instrumentation-processor` annotation processor from being reported as not used, if we want to apply it in `distribution.api-java`

Reply:
🤔 This might be good enough for now, but in the long run we need a way to "`@SuppressWarnings`" for @autonomousapps's dependency checker, and include certain dependencies across the board even when they are not used.

Reply:
Hi! Please file an issue and I'd be happy to take a look.

In my own work projects, I often have convention plugins that apply common `exclude()` rules. For example, if a dependency is itself added by a plugin, I also just suppress all warnings relating to that dependency in the same place.